messages/md.retrieve.json

```diff
@@ -0,0 +1,48 @@
+{
+  "retrieveCmd": {
+    "description": "retrieve metadata from an org using Metadata API\nUses Metadata API to retrieve a .zip of XML files that represent metadata from the targeted org. The default target username is the admin user for the default scratch org. You can retrieve and deploy up to 10,000 files or 400 MB (39 MB compressed) at one time.",
+    "examples": [
```

Comment:
Can we add the following brief explanations before each example?  This mirrors what Shane did with mdapi:deploy (please correct if my desc is wrong):

"Retrieve metadata using specified target directory:",
"Specify a manifest file:",
"Give a custom name to the retrieved .zip file and automatically extract its contents:",